src/client/reducers/reducer-learner.jsx

```diff
@@ -11,6 +16,22 @@ export default function(state = { loading: true }, action) {
       return {
         learners: state.learners,
         loading: action.loading,
+        skillSearch: state.skillSearch,
+        nameSearch: state.nameSearch,
+      };
+    case SEARCH_BY_SKILL:
+      return {
+        learners: state.learners,
+        loading: state.loading,
+        skillSearch: action.payload,
+        nameSearch: !action.payload,
```

Comment:
....so if it's a skill search, `nameSearch` is set to `false`? (and if it's a name search, `skillSearch` is set to `false`?) 

...what if there's some kind of bug and action.payload is empty? then if it's a skill search, won't `nameSearch` get set to `true`? does this have any unintended behavior? maybe investigate (and write unit tests) in a future story, approving the PR for now.